app/views/components/docs/contextual_guidance.yml

```diff
@@ -1,5 +1,16 @@
 name: Contextual guidance
 description: Provides a container with additional information when focusing an input field (e.g. input, textarea)
+body: |
+  This component is build to be used with a focusable elements that refers to it.
+
+  The contextual guidance must have an id. The input must refer the contextual guidance using a `data-contextual-guidance`
+  attribute (e.g. `<input data-contextual-guidance="my-contextual-guidance-id">`).
+
+  The guidance container is set to float on the page and prevent other elements from moving around (e.g. pushing next fields down the page).
```

Comment:
Might be worth clarifying this applies to desktop mode.

Reply:
Good all. Updated to clarify that applies to mobile and desktop only.